org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -148,6 +150,20 @@ public SpaceMode DistanceSpaceMode
         [Tooltip("Speed for extending the moving button visuals when selected by a non-touch source.")]
         private float extendSpeed = 0.5f;
 
+        /// <summary>
+        /// Flag that enables (true) or disables (false) the Canvas Rounded Rect on hover only.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Enables (true) or disables (false) the Canvas Rounded Rect on hover only.")]
+        public bool EnableOnHoverOnlyCanvasRoundedRect = false;
+
+        /// <summary>
+        /// Flag that enables (true) or disables (false) the Front plate Raw Image on hover only.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Enables (true) or disables (false) the Front plate Raw Image on hover only.")]
+        public bool EnableOnHoverOnlyFrontPlateRawImage = false;
```

Comment:
Although I don't think we yet have explicit coding guidelines for MRTK3, we've thus far enforced some similar rules to MRTK2, including "Fields should always be private, with public or protected property accessors."

https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk2/contributing/coding-guidelines?view=mrtkunity-2022-05#access-modifiers

Please update these fields to `private` and adding public fields pointing to them, if needed.

Reply:
You could also convert them into auto-properties and assign the attribute directly to the backing field with `[field: SerializeField]` :

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/1a61719eb59fa5042db916dbb031f8c3ff9eefe5/org.mixedrealitytoolkit.core/Interactables/StatefulInteractable.cs#L48-L50

Reply:
Thx for the feedback, I wasn't aware of those guidelines, I'll update accordingly.

Reply:
Done in latest commit.